plugin/completion.py

```diff
@@ -89,8 +89,12 @@ def epilogue(self, item: Dict[str, Any]) -> None:
             self.view.run_command("lsp_apply_document_edit", {'changes': edits})
         command = item.get("command")
         if command:
+            if command['command'] == "editor.action.triggerSuggest":
```

Comment:
I would rather see this check done in LspExecuteCommand itself. So that anyone who invokes that command doesn't have to bother with these checks.